showcase-app/main.js

```diff
@@ -84,9 +96,16 @@ const updateState = async () => {
     // Getting the name from the vcard if exists
     const name = getStringNoLocale(me, SCHEMA_INRUPT.name);
 
```

Comment:
@pheyvaer 
I assumed this is where the name was fetched from the IDP.
Is this SCHEMA_INRUPT.name property even necessary? Should it be fetched and displayed somewhere?

Reply:
It should be displayed once the user is logged in. That is the point of this issue/PR.

Reply:
ok, that's how it was implemented in this PR